core/services/log/broadcaster.go

```diff
@@ -548,15 +591,47 @@ func (b *broadcaster) MarkConsumed(lb Broadcast, qopts ...postgres.QOpt) error {
 	return b.orm.MarkBroadcastConsumed(lb.RawLog().BlockHash, lb.RawLog().BlockNumber, lb.RawLog().Index, lb.JobID(), qopts...)
 }
 
+// test only
```

Comment:
If they are test only I don't suppose they can be moved to `helpers_test.go`? Only works if they are used within `services/log` tests